utam-compiler/src/main/java/utam/compiler/grammar/UtamElement.java

```diff
@@ -193,7 +205,30 @@ private UtamComposeMethod buildWaitForComposeMethod() {
         null, null, null);
     List<UtamMethodAction> compose = Collections.singletonList(
         new UtamMethodActionWaitForElement(name));
-    return new UtamComposeMethod(methodName, description, compose);
+    return new UtamComposeMethod(methodName, description, compose, !isPrivateWait());
+  }
+
+  /**
+   * Element with "load" should not have parameters.
+   *
+   * @return
+   */
+  private void validateElementArgumentsForLoad() {
+    if ((selector != null && selector.hasArgsNode())
+            || (filter != null && !filter.getMatcherParameters().isEmpty()
+            || (traversal instanceof Container))) {
```

Comment:
I'm not sure if this condition returns true if parent element has args or filter, pls confirm with tests